cinema/views.py

```diff
@@ -127,6 +125,25 @@ def upload_image(self, request, pk=None):
 
         return Response(serializer.errors, status=status.HTTP_400_BAD_REQUEST)
```

Comment:
In the `upload_image` method, you're directly returning the serializer's errors in the response. While this is not wrong, it could lead to exposing internal details to the client. Consider wrapping these errors in a more user-friendly message.